lib/model/QubitInformationObject.php

```diff
@@ -1236,6 +1241,40 @@ function($row)
     return QubitPdo::fetchColumn($sql);
   }
 
+  /**
+   * Get the digital object's public URL
+   *
+   * @return string  digital object URL or null
+   */
+  public function getDigitalObjectPublicUrl()
+  {
+    require_once(dirname(__FILE__).'/../../vendor/symfony/lib/helper/UrlHelper.php');
```

Comment:
Is it possible to use something like: sfContext::getInstance()->getConfiguration()->loadHelpers('Url'); here?
Or alternatively if for some reason we can't do that in CLI tasks, can we try something like: require_once(sfConfig::get('sf_root_dir').DIRECTORY_SEPARATOR.'vendor/symfony/lib/helper/UrlHelper.php')

Just to avoid hard coding folder paths. In the off chance we ever moved tasks around under the tasks/ folder, it could break here.